<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Highscores</class>
 <widget class="QWidget" name="Highscores">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Highscores</string>
  </property>
  <widget class="QLabel" name="sizeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Size:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="smallButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Small</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="mediumButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Medium</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="largeButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Large</string>
   </property>
  </widget>
  <widget class="QListWidget" name="usernameWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>120</width>
     <height>175</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QListWidget" name="timeWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>120</width>
     <height>175</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>90</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Game Time:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>Highscores</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
